src/test/java/org/codeforamerica/shiba/pages/SubmitPageTest.java

```diff
@@ -31,6 +32,7 @@
         "spring.main.allow-bean-definition-overriding=true",
         "pagesConfig=pages-config/test-submit-page.yaml"
 })
+@Tag("integration")
```

Comment:
Given tests only need this tag when accessing the db, is there another way we could name them that would be more reflective of that?